tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/tester/ImputationTester.scala

```diff
@@ -49,12 +49,23 @@ class ImputationTester(testerConfig: ImputationConfig)(
     )
     val outputBaseName =
       imputationInputs.getBasename(workflowName)
+    val haplotypeDatabase =
+      imputationInputs.getHaplotypeDatabase(workflowName)
+    val splitOutputToSingleSample =
+      imputationInputs.getSplitOutputToSingleSample(workflowName)
+    val singleSampleVcfs =
+      imputationInputs.getSingleSampleVcfs(workflowName)
+    val singleSampleVcfsIndices =
+      imputationInputs.getSingleSampleVcfsIndices(workflowName)
     val resultsCloudPath =
       workflowTest.runParameters.resultsCloudPath
     val truthCloudPath = workflowTest.runParameters.truthCloudPath
 
-    // TODO add in optional single_sample_vcfs if necessary
     val validationInputs = ImputationValidationInputs(
+      haplotype_database = haplotypeDatabase,
```

Comment:
I think these also need to be added to the ImputationValidationInputs scala file as well